src/azul/plugins/metadata/anvil/__init__.py

```diff
@@ -285,20 +298,17 @@ def recurse(mapping: MetadataPlugin._FieldMapping, path: FieldPath):
                 elif isinstance(name_or_type, str):
                     if new_path == ('entity_id',):
                         pass
-                    elif new_path == ('contents', 'files', 'uuid'):
-                        # Request the injection of a file URL …
-                        result[path]['file_url'] = 'files.file_url'
-                        # … but suppress the columns for the fields …
-                        result[path][path_element] = None
-                    elif new_path == ('contents', 'files', 'version'):
-                        # … only used by that injection.
-                        result[path][path_element] = None
+                    elif new_path in custom_field_names:
+                        result[path][path_element] = custom_field_names[new_path]
                     else:
                         result[path][path_element] = name_or_type
                 else:
                     assert False, (path, path_element, name_or_type)
 
         recurse(self._field_mapping, ())
+        # Add a file URL column. It isn't an indexed field so isn't in the field
+        # mapping, rather it is dynamically added in during manifest generation.
```

Comment:
```suggestion
        # The file URL is synthesized from the `uuid` and `version` fields. Above, we already configured these two fields to be omitted from the manifest since they are not informative to the user.
```